src/main/java/ui/UI.java

```diff
@@ -174,6 +181,9 @@ private void showMainWindow(String repoId) {
         // Should only be called after panels have been initialized
         ensureSelectedPanelHasFocus();
         initialisePickers();
+        if (isFirstTimeUser && TestController.shouldOpenSampleBoard()){
+            openSampleBoard();
+        }
```

Comment:
May be this problem goes beyond this PR, but the dependency from this class to `TestController` should be in the other direction.